src/elektra/redshift.ini

```
@@ -0,0 +1,148 @@
+; Elektra spec file for redshift@490ba2aae9cfee097a88b6e2be98aeb1ce990050 (= redshift v1.12 + some small commits)
+; Based on: https://github.com/ElektraInitiative/libelektra/blob/master/examples/spec/lcdproc/LCDd.ini
+
+; TODO:
+; 6. add command line arguments to specification and split up colon-separated options for location-providers and adjustment-methods into separate CLI arguments.
+
+[]
+mountpoint = redshift.ecf
+infos/plugins = ni validation
```

ni is not recommended for normal configs, toml will be probably a good default:

```suggestion
infos/plugins = toml validation
```

---

Thanks!

After this change I am running into some issues. 
Do you have a suggestion on how to proceed? 

### Can't spec-mount
Unfortunately, after this change, I can't `spec-mount` anymore:

1. ``sudo kdb mount -v `pwd`/redshift.ini spec:/tests/redshift ni`` works fine
2. `sudo kdb spec-mount -v /tests/redshift` yields
```
Got mountpoint from spec:/tests/redshift with 23 nodes, configfile: redshift.ecf and mountpoint: /tests/redshift
The command kdb spec-mount terminated unsuccessfully with the info:
Too many plugins!
The plugin range can't be positioned at position presetstorage anymore.
Try to reduce the number of plugins!

Failed because presetstorage with 5 is larger than 4
```

Related issues are https://github.com/ElektraInitiative/libelektra/issues/2133 and https://github.com/ElektraInitiative/libelektra/issues/1274 . 
Do I have to give up one of the check plugins so only 4 plugin slots are used?

### Changing spec file to use toml
I thought I can have the spec file also in toml, so the ni plugin won't be loaded and less plugin slots are required.
However, if I change the spec file to be toml and use  
``sudo kdb mount -v `pwd`/redshift.ini spec:/tests/redshift toml`` 
there is a problem with the empty section `[]` at line 7. Empty sections aren't valid TOML syntax:
````
Sorry, module toml issued the error C03100:
Validation Syntactic: Line ~7: syntax error, unexpected BRACKETS_CLOSE
```

---

TOML loads base64+directoryvalue plugin, this might hit the maximum number of plugins. Simply leave the default plugin open for now:

```suggestion
```

(I also don't think that you manually need to load the validation plugin).

When Elektra starts using toml by default, the line will not be needed anymore. Until then, once https://github.com/ElektraInitiative/libelektra/pull/3693 gets fixed, toml can be used manually as argument to `spec-mount`.

---

Thanks, done.